src/OpenCvSharp/Modules/core/Enum/LogLevel.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace OpenCvSharp;
+
+/// <summary>
+/// cv::utils::logging::LogLevel
+/// </summary>
+public enum LogLevel
+{
+    /// <summary>
+    /// for using in setLogVevel() call
+    /// </summary>
+    LOG_LEVEL_SILENT = 0,
```

Comment:
I suggest removing the common prefix `LOG_LEVEL_` from each field name, which is the same as the type name.

```suggestion
    SILENT = 0,
```

Reply:
Thanks, that's a good point.